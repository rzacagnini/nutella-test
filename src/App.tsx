import React from 'react';

export const App = (): JSX.Element => {
  return <h1> Hello, world! </h1>;
};
