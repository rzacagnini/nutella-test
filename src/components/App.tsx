import React from 'react';

type AppParams = {
  message: string;
};

export const App = ({ message }: AppParams): JSX.Element => {
  return <h1> {message} </h1>;
};
