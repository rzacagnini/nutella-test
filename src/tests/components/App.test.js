import React from 'react';
import { render, cleanup } from '@testing-library/react';

import { App } from '../../components/App';

describe('App tests!', () => {
  it('renders', () => {
    const { asFragment } = render(<App message="Testing!" />);
    expect(asFragment()).toMatchSnapshot();
  });
  afterAll(cleanup);
});
