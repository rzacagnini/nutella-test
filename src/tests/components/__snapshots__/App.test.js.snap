// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests! renders 1`] = `
<DocumentFragment>
  <h1>
     Testing! 
  </h1>
</DocumentFragment>
`;
